assets/xml/objects/object_gj.xml

```diff
@@ -0,0 +1,23 @@
+<Root>
+    <File Name="object_gj" Segment="6">
+        <DList Name="gGjDL_000DC0" Offset="0xDC0"/>
```

Comment:
Although its not required I'd suggest trying to name these, even if its just `gGanonsCastleRubble1DL`, `gGanonsCastleRubble2DL`, and so on.